kasa/__init__.py

```diff
@@ -13,9 +13,14 @@
 """
 from importlib.metadata import version
 
+from kasa.credentials import Credentials
 from kasa.discover import Discover
 from kasa.emeterstatus import EmeterStatus
-from kasa.exceptions import SmartDeviceException
+from kasa.exceptions import (
+    SmartDeviceAuthenticationException,
```

Comment:
```suggestion
    AuthenticationException,
```
Let's keep it simple, the unfortunate naming with "Smart" cam from the predecessor of this lib and was kept for compat reasons.

Reply:
Done